samples/exemplar/src/test/kotlin/com/squareup/exemplar/RedisRateLimitedActionTests.kt

```diff
@@ -19,6 +20,7 @@ class RedisRateLimitedActionTests : AbstractRateLimitedActionTests() {
     override fun configure() {
       install(ExemplarTestModule())
       install(RedisBucket4jRateLimiterModule(DockerRedis.config, JedisPoolConfig(), useSsl = false))
+      install(RedisFlushModule())
```

Comment:
```suggestion
      install(RedisTestingModule())
```

I lean toward making it explicit in the module name that this should be installed for testing only. It also maps in functionality to JdbcTestingModule which includes similarly TruncateTablesService.

Reply:
There's already a `RedisTestModule` that installs the `FakeRedis` in-memory redis. Do you think the similar naming would cause confusion? Could just add doc comments